.circleci/config.yml

```diff
@@ -35,6 +35,7 @@ jobs:
             cd ~/test_application_js
             npm install
             npm test
+            npm browser-test
```

Comment:
should be `npm run browser-test` since `browser-test` is just a script in `package.json`